src/ftw_cli/cli.py

```diff
@@ -106,13 +108,14 @@ def inference_run(input, model, out, resize_factor, gpu, patch_size, batch_size,
 
 @inference.command("polygonize", help="Polygonize the output from inference for the raster image given via INPUT.")
 @click.argument('input', type=click.Path(exists=True), required=True)
-@click.option('--out', '-o', type=str, required=True, help="Output filename for the polygonized data.")
+@click.option('--out', '-o', type=str, required=True, help="Output filename for the polygonized data. " + SUPPORTED_POLY_FORMATS_TXT)
 @click.option('--simplify', type=float, default=None, help="Simplification factor to use when polygonizing.")
```

Comment:
```suggestion
@click.option('--simplify', type=float, default=15, help="Simplification factor to use when polygonizing. Assumes meters, default is 15.")
```

Reply:
Is this actually always meters or does it depend on the CRS unit?
Do we have any evidence on the 15?

Reply:
> Is this actually always meters or does it depend on the CRS unit?

Depends on the CRS unit. But we only have S2 as input right now. So I lean towards assuming that in our code - but would be better for now if we raised an error if the projection was not in meters, and say that's not supported. Vs having the user have to understand the nature of their CRS to supply a simplify parameter. I guess we also could just translate meters to other CRS's. I'd very much like this call to assume meters - I don't like the simplify calls in the geo world that force you to know / look up what units your CRS is in.

Reply:
The area computation has been improved to work on any CRS. If we restrict to m, we could potentially also remove that code.

On the other hand, I feel like it's fine to just elaborate in the description. I mean it's still meters in many cases, including S2. Translating meters to other CRSes is not that easy.

Reply:
See 2fa233ed